### DISCUSSION TITLE
Support for MY92XX-based LED drivers

### DISCUSSION THREAD

**Discussion by dillydylann** [RavenSystem/esp-homekit-devices#2157] [New Features]
I have a few lightbulbs that utilize the [MY9291 driver chip](https://tinkerman.cat/post/ailight-hackable-rgbw-light-bulb/) that I would like to use with this firmware but still don't have support for it.

After looking on here, there's been some discussion about adding support for MY92XX-based drivers (latest being from March 2021):
- https://github.com/RavenSystem/esp-homekit-devices/issues/70
- https://github.com/RavenSystem/esp-homekit-devices/issues/368
- https://github.com/RavenSystem/esp-homekit-devices/issues/1150
- https://github.com/RavenSystem/esp-homekit-devices/pull/1156
- https://github.com/RavenSystem/esp-homekit-devices/pull/1369

And there's some references in code for the MY92XX:
https://github.com/RavenSystem/esp-homekit-devices/blob/eb2eec94a44db6e17f2e411d315e29dc50ae345a/HAA/HAA_Main/main/header.h#L336
https://github.com/RavenSystem/esp-homekit-devices/blob/eb2eec94a44db6e17f2e411d315e29dc50ae345a/HAA/HAA_Main/main/main.c#L3584-L3588

Has there been any progress with this or has it been scrapped?

**Comment by RavenSystem** [comment]
No progress, sorry. I have not any MY92XX, and currently there is not enough free space in firmware to fit needed functions.
